web/impact/impact/views/general_view_set.py

```diff
@@ -22,9 +27,12 @@ def model(self):
         model = snake_to_model_name(self.kwargs.get('model', ''))
         related_model = self.kwargs.get('related_model', '')
         model_name = model_name_case(model, related_model)
-        return apps.get_model(
+        model = apps.get_model(
             app_label=self.kwargs['app'],
             model_name=model_name)
+        if model.__name__ in EXCLUDE_MODELS:
+            raise Http404
```

Comment:
@mkiterian why do we raise a 404? Why not raise a `LookupError` just like the `get_model` method? https://github.com/django/django/blob/402c0caa851e265410fbcaa55318f22d2bf22ee2/django/apps/config.py#L180

Lastly, why do we exclude the model from both the `GeneralViewSet` and the `urls.py`, are the endpoints generated twice?

Reply:
The exclusion in urls.py is to make sure that it's not listed here: `http://localhost:8000/api/accelerator/`